### DISCUSSION TITLE
Latrines

### DISCUSSION THREAD

**Discussion by Mephisto-Fibs** [Keriew/augustus#423] [Ideas]
Romans have bowl movements, too. Thats why public latrines date back to the 2nd century BC. Whether intentionally or not, they where places to socialise and offered a great opportunity finalyze business transactions. 

Latrines could be a new type of health building - it is extremely undesirable to live nearby but offers beneficial effects. Of course, it would need water coverage. 

The Latrine is where the Romans sit down, have a relaxed chat and do business with each other. Therefore, the existence of the latrine building offers, say, 5 % better prices for all goods depending on latrine coverage in the city. 

New message when the first latrine is active: 
"pass the xylospongium to me, fellow citizen and let's seal this deal" :D

**Comment by gianlore** [comment]
Penso che l'edificio "Bagni"  o "Barbiere" abbia la stessa funzione

**Reply by Purakane** [reply]
me likey

**Reply by PaxBotanica** [reply]
I would really like to see public and private latrines. I think public latrines should be a bath-style health requirement for insulae development and private latrines required for patricians. This would add an element of historical accuracy thats missing atm.
